src/app/psat/explore-opportunities/explore-opportunities-form/system-data-form/system-data-form.component.html

```diff
@@ -29,11 +28,11 @@
 
         </div>
     </form>
-    <form [formGroup]="modificationForm">
-        <div class="form-group"
+    <form [formGroup]="modificationForm" class="explore-opps-form">
+        <div class="form-group explore-opps-formgroup"
             [ngClass]="{'invalid': modificationForm.controls.flowRate.invalid, 'error': modificationWarnings.flowError !== null}">
-            <label for="modificationFlowRate">Modification Flow Rate</label>
-            <div class="input-group">
+            <label for="modificationFlowRate" class="mod-value-label">Modification Flow Rate</label>
+            <div class="input-group modValueGroup__inputGroup">
```

Comment:
`modValueGroup_inputGroup` class is in system-data-form.component.html (3) and in variable-frequency-drive-form.component.html (2)